tools/integration_tests/read_large_files/read_one_large_file_sequentially_test.go

```diff
@@ -18,7 +18,6 @@ import (
 	"bytes"
```

Comment:
I would suggest to change name of this file also, to something like seq_read_large_file_test.go